source/includes/api/_widget.md

```diff
@@ -582,281 +595,543 @@ curl -X GET https://api.resourcewatch.org/v1/widget?includes=metadata
-name         |           Name of the widget            |    Text |                                        Any Text |      Yes
-description  |       Description of the dataset        |    Text |                                        Any text |       No
-source       |     Publisher of the original code      |    Text |                                        Any text |       No
-sourceUrl    |         Link to publisher page          |    Text |                                         Any url |       No
-application  | Application to which the widget belongs |   Array | gfw, forest-atlas, rw, prep, aqueduct, data4sdg |      Yes
-authors      |           Name of the authors           |    Text |                                        Any text |       No
-queryUrl     |  Url with the data of the chart shows   |    Text |                                 Any valid query |       No
-widgetConfig |           Vega configuration            |  Object |                                    Valid object |       No
-published    |        If it's available to use         | Boolean |                                    true - false |       No
-freeze       |        If the data is frozen            | Boolean |                                    true - false |       No
-protected    |        If it's a protected widget (not is possible to delete if it's true)         | Boolean |   true - false |       No
-verified     |     If it's verified by other user      | Boolean |                                    true - false |       No
-template     |           If it's a template            | Boolean |                                    true - false |       No
-default      |       If it's default for dataset       | Boolean |                                    true - false |       No
-layerId      |   UuId of the relationship with layer   |    Text |                                   Uuid of layer |       No
-dataset      |           UuId of the dataset           |    Text |                                 Uuid of Dataset |       No
-env         |                                            Environment of the Widget. Set to 'production' by default                                               |   String |                                 Valid string |       No
+This overwrite will only happen if a previous value already existed for the respective field. Also, keep in mind that these changes will NOT be persisted to the database, and will only affect the current request's response. You can combine these custom query parameters with an [overwritten query url](#overwrite-the-query-url-of-a-widget-response), as seen on the included example.
 
-It is possible to create a widget that has a different `env` property to its parent dataset.
 
-> To create a widget, you have to do a POST request with the following body:
+### Include entities associated with the widget
```

Comment:
This section is duplicated, no?

Reply:
we have one above for the "get multiple widgets" endpoint. this one is under the "get a single widget by id" endpoint, and it just points to the other one. The idea is to keep the docs DRY, but at the same time cover users who go straight for the "get single widget by id" without reading what came before.

Reply:
Ah okay. I would maybe just say something like, "As with getting all widgets, you can include related entities for a single widget," and then link. Not a big deal though.